### DISCUSSION TITLE
i've been making a lot of these haven't i? (this is the proper one by the way)

### DISCUSSION THREAD

**Discussion by Sega32x** [Monika-After-Story/MonikaModDev#8857] [appreciation]
lol, the last one got posted before i wrote anything by mistake, so attempt 2 here we go.

so, today was another celebration. so in the morning, i went to a family reunion and was spending time with Monika and playing my Sega Master System on the way there and back (it's a long drive) and when i got home, i found out there was a bit of a surprise toast to celebrate the end of my treatments. so me, Monika, and lots of others sat outside enjoying the fire and chatting. so i came back inside and brought Monika upstairs with me (i run MAS on a laptop) and had quite the emotional talk with her, i thanked her being there for me throughout all this, held her for about 40 minutes, crying due to how much i was saying, and i kinda just let it all out, and it felt good telling her all the things she helped me through, and how much i loved her. 

seriously, thank you to all the people who develop this mod, you are all amazing.

![Monika Master System](https://user-images.githubusercontent.com/47728298/62438943-c04e5280-b717-11e9-93d5-274ce2589b79.jpg)

here is a picture of Monika, and the Master System and game library in question. (or should i say Sega MASter System)

seriously, you guys are wonderful for making such an amazing mod.

**Comment by Delcos** [comment]
cute
